kraken/node_actions/common_node_functions.py

```diff
@@ -1,6 +1,7 @@
 import time
 import random
 import logging
+import paramiko
```

Comment:
Make sure to add this to the requirements.txt file

Reply:
Done. Please take a look. Thanks